content/blog/2022/2022-05-12-Google-Tides.md

```diff
@@ -0,0 +1,30 @@
++++
+author = "Zala Gruden"
+date = "2022-05-12"
+draft = false
+title = "We have launched a new project on the topic of AI in schools"
+type = "blog"
+thumbImage = "/blog_img/2022/2022-05-12-google-tides.png"
+frontPageImage = "/blog_img/2022/2022-05-12-google-tides.png"
+blog = ["Google", "Tides Foundation", "project", "visualization"]
+shortExcerpt = "Find out more about our new project about AI in Schools."
+longExcerpt = "The website to our latest project has just been launche. Read more about the project in the blogpost and Check out the site to find out more about the prepaired activities."
```

Comment:
> The website to our latest project has just been launche.
Delete above. Keep only:
Read more about the project in the blog post and check out the site to find out more about the prepared activities.